packages/medusa/src/api/routes/admin/orders/get-reservations.ts

```diff
@@ -40,7 +40,7 @@ import { extendedFindParamsMixin } from "../../../../types/common"
  *     content:
  *       application/json:
  *         schema:
- *           $ref: "#/components/schemas/AdminGetReservationReservationsReq"
+ *           $ref: "#/components/schemas/AdminReservationsRes"
```

Comment:
This should be `AdminReservationsListRes`

Reply:
Good catch 🎣 . Done in efda8f4695e084de83426d45b33a56af704b15e4